docs/contacts.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Contacts and Mailing Lists
+nav_order: 300
```

Comment:
This will make it go after "Downloading ICU" and before "ICU User Guide" so that it is adjacent to high level info and doesn't look like it is inside the User Guide.

```suggestion
nav_order: 150
```